### ISSUE TITLE
Page does not load on Mac+Safari

### ISSUE THREAD

**Issue by Caseysch** [sb/smallbasic-editor#127]
macOS Catalina, version 10.15.3
Safari version 13.0.5

Attempting to code online results in a loading spinner and then a blank page. Complete console log:

```
[Info] Blazor is running in dev mode without IL stripping. To make the bundle size significantly smaller, publish the application or see https://go.microsoft.com/fwlink/?linkid=870414 (blazor.webassembly.js, line 1)
[Log] WASM: Initialized (mono.js, line 1)
[Error] Unhandled Promise Rejection: RangeError: Maximum call stack size exceeded.
	(anonymous function) (blazor.webassembly.js:1:37752)
	i (blazor.webassembly.js:1:36542)
	promiseReactionJob
```

We should either fix this, or detect failure to load and provide a link to the old version of the online editor or prompt them to switch browsers.

**Comment by Caseysch** [comment]
https://github.com/mono/mono/issues/18646 maybe?

**Comment by Caseysch** [comment]
No longer seeing this specific issue.

Seeing a browser crash with limited reproducibility, but it looks like this was fixed.

**Comment by nonkit** [comment]
I also confirmed that #127 fixed with Safari 13.1.1 (15609.2.9.1.2) on macOS Catalina 10.15.5.  Thanks.
